libraries/botbuilder-dialogs/package.json

```diff
@@ -26,6 +26,7 @@
     "@microsoft/recognizers-text-suite": "1.1.4",
     "@types/node": "^10.12.18",
     "botbuilder-core": "4.1.6",
+    "botframework-schema": "../botframework-schema",
```

Comment:
This isn't necessary as botbuilder-core surfaces the contents of `botframework-schema`.